### DISCUSSION TITLE
Recording data in h5 data container via python API?

### DISCUSSION THREAD

**Discussion by matijazigic** [ifm/ifm3d#439] [Q&A]
Hello again :),

is there python API method for recording data or some code snippet to do saving of data in exact same way as described [here](https://ifm3d.com/latest/ODS/Recording/ods_data_recording.html) using iVA? For example, to record one frame of data, which can be used later for extrinsic calibration?

**Comment by desengph** [comment]
Hello @matijazigic 
We have an internal project that can be used for this purpose. 
Unfortunately, it has not been released yet because it only works on mid to high-end devices where the data is stored fast enough to not build up internal buffers. A standard VPU device running an ODS app instance will not be able to handle the additional load. 

I can email you a link to download this project if you're interested in trying it out.
